### PR TITLE
Jhkim/accessbility tab order

### PR DESCRIPTION
**Pull request by jhkimnew**
1. Fixed tab order for Filter button
2. Adjust color value to enhance color ratio

### CODE REVIEW

File: src/app/files/navigation.component.ts [hunk 1]
```diff
@@ -18,15 +18,15 @@ export class Drop {
...
+                <ul aria-label="Hit the space key to select" tabindex="0" *ngIf="_crumbs.length > 0" [hidden]="_typing" class="nav border-color" (keyup.space)="onClickAddress($event)" (keyup.enter)="onClickAddress($event)" (click)="onClickAddress($event)">
                     <li *ngFor="let item of _crumbs; let i = index;" 
                         (dragover)="dragOver(i, $event)" 
                         (drop)="onDrop(i, $event)"
                         (dragleave)="_selected=-1">
                         <span class="crumb hover-active" [class.background-active]="i==_selected" (click)="onClickCrumb(i, $event)">{{item}}</span>/
                     </li>
-                </ul>
-                <input #addressBar type="text" class="form-control" *ngIf="_crumbs.length > 0" [hidden]="!_typing" [ngModel]="_path" (ngModelChange)="onPathChanged($event)" throttle  />
+                </ul>
```
Comment (shhsu) [inline src/app/files/navigation.component.ts:28]: nit: extra space

File: src/app/files/navigation.component.ts [hunk 2]
```diff
@@ -18,15 +18,15 @@ export class Drop {
         <div>
             <button class="no-border pull-left color-active" title="Go Up" (click)="onClickUp()"><i class="fa fa-level-up"></i></button>
             <div class="fill">
-                <ul tabindex="0" *ngIf="_crumbs.length > 0" [hidden]="_typing" class="nav border-color" (keyup.space)="onClickAddress($event)" (keyup.enter)="onClickAddress($event)" (click)="onClickAddress($event)">
+                <ul aria-label="Hit the space key to select" tabindex="0" *ngIf="_crumbs.length > 0" [hidden]="_typing" class="nav border-color" (keyup.space)="onClickAddress($event)" (keyup.enter)="onClickAddress($event)" (click)="onClickAddress($event)">
```
Comment (shhsu) [inline src/app/files/navigation.component.ts]: Press space key?

File: src/app/files/navigation.component.ts [hunk 3]
```diff
@@ -18,15 +18,15 @@ export class Drop {
...
                     <li *ngFor="let item of _crumbs; let i = index;" 
                         (dragover)="dragOver(i, $event)" 
                         (drop)="onDrop(i, $event)"
                         (dragleave)="_selected=-1">
                         <span class="crumb hover-active" [class.background-active]="i==_selected" (click)="onClickCrumb(i, $event)">{{item}}</span>/
                     </li>
-                </ul>
-                <input #addressBar type="text" class="form-control" *ngIf="_crumbs.length > 0" [hidden]="!_typing" [ngModel]="_path" (ngModelChange)="onPathChanged($event)" throttle  />
+                </ul> 
+            <input aria-label="Enter a Path to go" title="Path Breadcrumbs" #addressBar type="text" class="form-control" *ngIf="_crumbs.length > 0" [hidden]="!_typing" [ngModel]="_path" (ngModelChange)="onPathChanged($event)" throttle  />
```
Comment (shhsu) [inline src/app/files/navigation.component.ts]: Enter breadcrumbs path?

File: src/app/main/get.component.ts [hunk 4]
```diff
@@ -55,7 +55,7 @@ import { HttpFacade } from 'common/http-facade';
         }
 
         button {
-          width: 100px;  
+            width: 100px;
```
Comment (shhsu) [inline src/app/main/get.component.ts:58]: nit: spacing

File: src/app/runtime/wac/components/install.component.ts [hunk 5]
```diff
@@ -38,7 +38,7 @@ import { PowerShellScripts } from "../../../../generated/powershell-scripts"
     }
 
     button {
-      width: 100px;  
+        width: 100px;
```
Comment (shhsu) [inline src/app/runtime/wac/components/install.component.ts:41]: spacing

**Review by shhsu — CHANGES_REQUESTED** [review]

File: src/app/files/navigation.component.ts (hunk 2 above)
Comment (jhkimnew) [reply src/app/files/navigation.component.ts]: good catch. Replaced with spacebar.

File: src/app/files/navigation.component.ts (hunk 3 above)
Comment (jhkimnew) [reply src/app/files/navigation.component.ts]: Updated. One more change. The newly added title property is not actually required. So, I removed it.

File: src/app/runtime/wac/components/install.component.ts (hunk 5 above)
Comment (jhkimnew) [reply src/app/runtime/wac/components/install.component.ts:41]: Fixed.

**Review by shhsu — APPROVED** [review]

**Pull request merged**